### PR TITLE
Fix/13inch width

### PR DESCRIPTION
**Pull request by WhiteNik16**

### CODE REVIEW

File: src/components/AppBar/AppBar.jsx [hunk 1]
```diff
@@ -9,12 +9,14 @@ import { Link } from 'react-router-dom'
 import { Icon } from '@/components'
 import { CONFIG } from '@/config'
 import { ROUTES_PATHS } from '@/const'
+import { useAppContext } from '../../hooks'
```
Comment (napalmpapalam) [inline src/components/AppBar/AppBar.jsx]: fix import path

File: src/components/AppBar/AppBar.jsx [hunk 2]
```diff
@@ -40,10 +42,17 @@ const AppBar = () => {
...
+              <Icon
+                iconClass='app-bar__logo-img'
+                idIcon={isDark ? 'icon-app-logo-dark' : 'icon-app-logo-light'}
+              />
+            ) : (
+              <Icon
+                iconClass='app-bar__logo-img'
+                idIcon={isDark ? 'icon-logo-mobile-light' : 'icon-logo-mobile-dark'}
+              />
+            )}
```
Comment (napalmpapalam) [inline src/components/AppBar/AppBar.jsx]: made idIcon variable with use memo instead of this ternary

**Review by napalmpapalam — CHANGES_REQUESTED** [review]

File: src/components/AppBar/AppBar.jsx (hunk 1 above)
Comment (WhiteNik16) [reply src/components/AppBar/AppBar.jsx]: fixed

File: src/components/AppBar/AppBar.jsx (hunk 2 above)
Comment (WhiteNik16) [reply src/components/AppBar/AppBar.jsx]: fixed

**Pull request merged**